.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[ci skip]')"
     strategy:
       matrix:
-        node-version: [10.19, 12.16, 12.19, 14]
+        node-version: [14, 16, 18]
```

Comment:
i think some of our internal code still relies on [node12](https://git.corp.adobe.com/nui/cli/blob/master/.circleci/config.yml#L88). Is it possible to keep node12 until we phase that out everywhere internally?
(if not, please make sure this is a breaking release)

Reply:
Also - usually we pick the exact node versions I/O Runtime is using, not node:latest.